dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.org.eclipse.microprofile.rest.client-2.0.feature

```diff
@@ -0,0 +1,11 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.org.eclipse.microprofile.rest.client-1.4
+singleton=true
+-features=com.ibm.websphere.appserver.javax.cdi-2.0, \
+ com.ibm.websphere.appserver.javax.annotation-1.3, \
+ com.ibm.websphere.appserver.javax.jaxrs-2.1, \
+ com.ibm.websphere.appserver.org.eclipse.microprofile.config-2.0
```

Comment:
That isn't the right feature name. Should be `io.openliberty.org.eclipse.microprofile.config-2.0`